### DISCUSSION TITLE
Syncing obsidian with Github on Windows

### DISCUSSION THREAD

**Discussion by Gaaruda** [BryanJenksCommunity/FAQ#39] [Q&A] [answered]
I had seen your video on how to sync obsidian vault with git and git hub
I am on windows.
I have successfully made repositories and pushed my data in github.
But unable to figure out how to automatically sync it to github. Can you Please Help.

**Comment by tallguyjenks** [comment, marked as answer]
From my original forum post on the git workflow:

https://forum.obsidian.md/t/obsidian-github-integration-for-sync-and-version-control/6369

This was what someone suggested for windows:

https://github.com/simonthum/git-sync

one could also simply write a `.bat` file and put it into the task scheduler

**Comment by Gaaruda** [comment]
Thanks
